python/kmeans.py

```diff
@@ -0,0 +1,48 @@
+import time
+from typing import List, Tuple
+
+import mgp
+
+from sklearn.cluster import KMeans
+from sklearn.preprocessing import StandardScaler
+
+
+def get_labels(nodes, embeddings, number_of_clusters):
+    classes_embeddings_classes_dict, classes_embeddings_list = get_groups(number_of_clusters, embeddings, nodes)
```

Comment:
If `get_groups` only returns a `classes_embedding_list`, why does this line unpack its return value to two variables?